src/function/table.rs

```diff
@@ -124,13 +124,6 @@ impl Table {
         Some(&self.vals[off].1)
     }
 
-    pub(crate) fn get_mut(&mut self, inputs: &[Value]) -> Option<&mut TupleOutput> {
```

Comment:
`get_mut()` is unsafe because it does not require the caller to update the timestamp. Use `insert_and_merge` instead.